bigbluebutton-html5/imports/ui/components/about/component.jsx

```diff
@@ -34,17 +34,27 @@ const intlMessages = defineMessages({
   },
 });
 
-const AboutComponent = ({ intl, clientBuild, copyright }) => (
-  <Modal
-    title={intl.formatMessage(intlMessages.title)}
-    dismiss={{
-      label: intl.formatMessage(intlMessages.dismissLabel),
-      description: intl.formatMessage(intlMessages.dismissDesc),
-    }}
-  >
-    {`${intl.formatMessage(intlMessages.copyright)} ${copyright}`} <br />
-    {`${intl.formatMessage(intlMessages.version)} ${clientBuild}`}
-  </Modal>
-);
+class AboutComponent extends PureComponent {
```

Comment:
Why are you changing pure functions to PureComponents everywhere?

Reply:
it was when I was trying to pass in the mountModal, i'll switch them back to how they were previously, since we didn't need it.